src/main/java/edu/cornell/kfs/tax/batch/TransactionOverrideCsvBatchInputFileType.java

```diff
@@ -0,0 +1,443 @@
+        return transactionOverrides;
+    }
+
+    /**
+     * Transforms each parsed line into a TransactionOverride BO if content is valid,
+     * or into a String representation of the parsed line if content is invalid.
+     * Assumes that TransactionOverrideCsv is the header enum.
+     * 
+     * @param parsedLine The line to process, as a Map from header names to values.
+     * @param lineNumber The current line number, starting from 1.
+     * @param docNumberMaxLength The max length of the BO's document number property.
+     * @param boxNumberMaxLength The max length of the BO's tax box property.
+     * @return A TransactionOverride object if a valid line, otherwise a tab-delimited String representation of the line.
+     */
+    protected Object convertParsedLineToVO(Map<String,String> parsedLine, int lineNumber, int docNumberMaxLength, int boxNumberMaxLength) {
+        boolean valid = true;
+        TransactionOverride transOverride = new TransactionOverride();
+        
+        // Verify that exactly one type of tax box was specified, and that the tax box has the expected max length. Also setup tax box properties if valid.
+        String box1099 = parsedLine.get(TransactionOverrideCsv.Form_1099_Box.toString());
+        String box1042S = parsedLine.get(TransactionOverrideCsv.Form_1042S_Box.toString());
+        if (StringUtils.isBlank(box1099)) {
+            if (StringUtils.isBlank(box1042S)) {
+                LOG.error("Found a line that does not specify a 1099 or 1042S tax box override. Line number: " + Integer.toString(lineNumber));
+                valid = false;
+            } else if (box1042S.length() > boxNumberMaxLength) {
+                LOG.error("Found a line with a 1042S box number that is too long. Line number: " + Integer.toString(lineNumber));
+                valid = false;
+            } else {
+                // Validation succeeded; configure override of a 1042S tax box.
+                transOverride.setTaxType(CUTaxConstants.TAX_TYPE_1042S);
+                transOverride.setBoxNumber(NULL_STRING.equalsIgnoreCase(box1042S) ? CUTaxConstants.TAX_1042S_UNKNOWN_BOX_KEY : box1042S);
+            }
+        } else if (StringUtils.isNotBlank(box1042S)) {
+            LOG.error("Found a line that specifies both a 1099 and 1042S tax box override. Line number: " + Integer.toString(lineNumber));
+            valid = false;
+        } else if (box1099.length() > CUTaxConstants.TAX_1099_MAX_BUCKET_LENGTH && !NULL_STRING.equalsIgnoreCase(box1099)) {
+            LOG.error("Found a line with a 1099 box number that is too long. Line number: " + Integer.toString(lineNumber));
+            valid = false;
+        } else {
+            // Validation succeeded; configure override of a 1099 tax box.
+            transOverride.setTaxType(CUTaxConstants.TAX_TYPE_1099);
+            transOverride.setBoxNumber(NULL_STRING.equalsIgnoreCase(box1099) ? CUTaxConstants.TAX_1099_UNKNOWN_BOX_KEY : box1099);
+        }
+        

```

Comment:
We want to minimize the labor intensity of the process for the users. Your implementation and explanation indicates that.